musicbrainz/csv/convert_to_csv.py

```diff
@@ -0,0 +1,131 @@
+# This file converts a JSON Lines file from MusicBrainz to a CSV file
```

Comment:
This probably should be a docstring (see [here](https://peps.python.org/pep-0257/))

Reply:
A docstring for the extract function?

Reply:
Or for the module... so you could put the docstring for the module as the first thing in the file. Since you have the comments here, that would be the most similar thing.